.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: publish
+
+on:
+  push:
+    paths:
+      - 'Bmx.CommandLine/**'
+      - 'Bmx.Core/**'
+      - 'Bmx.Idp.Okta/**'
+      - 'Bmx.Service.Aws/**'
+      - '.github/workflows/publish.yml'
+    branches:
+      - main
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        machine: [windows-latest, macos-latest, ubuntu-20.04]
+        architecture: [x64, arm64]
+        include:
```

Comment:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs
I think this should work

Reply:
I picked older ubuntu because the articles mentioned that linux versions older than the one ran to build will not be compatible.